ReactWindows/ReactNative.Shared/UIManager/DependencyObjectExtensions.cs

```diff
@@ -136,35 +138,37 @@ public static bool HasTag(this DependencyObject view)
             return s_properties.TryGetValue(view, out elementData) && elementData.Tag.HasValue;
         }
 
-        internal static void SetReactContext(this DependencyObject view, ThemedReactContext context)
+        internal static void SetEventEmitter(this DependencyObject view, IEventEmitter eventEmitter)
```

Comment:
The dependency object extensions were providing access to the ReactContext object for the sole purpose of gaining access to the EventDispatcher...  This change reduces the surface available to code accessing these extensions...